File: VB/MDIDemo.Module/Module.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic

Imports DevExpress.ExpressApp
Imports System.Reflection


Namespace MDIDemo.Module
    Public NotInheritable Partial Class MDIDemoModule
        Inherits ModuleBase
        Public Sub New()
            InitializeComponent()
        End Sub
        Protected Overrides Function GetAdditionalBusinessClassAssemblies() As IList(Of System.Reflection.Assembly)
            Return New System.Reflection.Assembly() { GetType(DevExpress.Persistent.BaseImpl.Person).Assembly }
        End Function
    End Class
End Namespace

File: CS/MDIDemo/MDIDemo.Module/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using System.Reflection;


namespace MDIDemo.Module
{
    public sealed partial class MDIDemoModule : ModuleBase
    {
        public MDIDemoModule()
        {
            InitializeComponent();
        }
        protected override IList<System.Reflection.Assembly> GetAdditionalBusinessClassAssemblies()
        {
            return new System.Reflection.Assembly[] { 
				typeof(DevExpress.Persistent.BaseImpl.Person).Assembly };
        }
    }
}
